web/ui/mantine-ui/src/components/ThemeSelector.tsx

```diff
@@ -30,11 +30,11 @@ export const ThemeSelector: FC = () => {
       }
     >
       {colorScheme === "light" ? (
-        <IconMoonFilled {...iconProps} />
+        <IconSunFilled {...iconProps} />
```

Comment:
By curiosity, why did you change the order of the Icon ?

Reply:
ok it was a stupid question I didn't see the condition about the colorScheme.

Reply:
Wait, but this does change it relative to my PR, right? Was that intended?

Reply:
I originally also had the icons the way around that @amanycodes changed it to, but then I changed it back to what I had in #16000 because the common thing to do on other websites for single-button switches seems to be to show the icon of the theme you are switching to, not the one that is currently active.

Cases in point:

* https://mantine.dev/
* https://react.dev/

I've seen some other examples that look like a single button, but they open a dropdown menu, and those show the icon of the currently selected theme:

* https://orm.drizzle.team/
* https://web.dev/

Probably doing it like that with a dropdown instead of a single-button three-way toggle is the best overall thing to do.

Reply:
ah crap, sorry about that @juliusv.

I can see other website where actually the icon you are seeing is the one matching the current theme. So I guess there is no big convention.

But yeah to avoid such confusion I guess the dropdown is the best way, but clicking on a button just to switch the theme is really smooth also